### PR TITLE
KOGITO-741: Bring Octokit support to Online Editor

### PR DESCRIPTION
**Pull request by jesuino**
Uses octokit to fetch github files in online editor.

### CODE REVIEW

**Comment by tiagobento** [comment]
Looks like there's an issue with CORS when trying to access GitHub's API. Will merge when it's fixed.

File: packages/online-editor/src/common/GithubService.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Octokit from "@octokit/rest";
+
+export class GithubService {
```
Comment (domhanak) [inline packages/online-editor/src/common/GithubService.ts:25]: A test for this class would be useful

**Comment by jesuino** [comment]
added a few tests @domhanak

File: packages/online-editor/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 202 Red Hat, Inc. and/or its affiliates.
```
Comment (tiagobento) [inline packages/online-editor/jest.config.js]: Wrong copyright

**Review by tiagobento — CHANGES_REQUESTED** [review]
Looks good to me, great work @jesuino! Please make sure that your code is properly formatted and the imports are cleaned up. Also, please re-check the copyrights in new files. Thx!

File: packages/online-editor/jest.config.js (hunk 2 above)
Comment (jesuino) [reply packages/online-editor/jest.config.js]: fixed, thanks @tiagobento

**Comment by tiagobento** [comment]
Removed the `do not merge` label since we solved the CORS issue with another HTTP header.

**Review by tiagobento — APPROVED** [review]

**Pull request merged**